openfisca_uk/initial_setup.py

```diff
@@ -48,6 +49,9 @@ def main():
                 print(
                     f"You currently have the synthetic FRS dataset for: {', '.join(map(str, SynthFRS.years))}."
                 )
+            with open(REPO / "tools" / "datasets.yml") as f:
```

Comment:
Is this so that it updates previous installations?

Reply:
It's actually just to have a stored file keeping track of the default dataset